include/sancus_support/timer.h

```diff
@@ -1,13 +1,19 @@
 #ifndef SANCUS_SUPPORT_TIMER_H
 #define SANCUS_SUPPORT_TIMER_H
 #include <msp430.h>
+#include <stdint.h>
 
-#define TACTL_DISABLE       (TACLR)
-#define TACTL_ENABLE        (TASSEL_2 + MC_1 + TAIE)
-#define TACTL_CONTINUOUS    ((TASSEL_2 + MC_2) & ~TAIE)
+#define TACTL_DISABLE       (TACLR) // 0x04
```

Comment:
it should be possible to include these symbolic constants in your inline asm as "constant" contraint parameter ("i" or something I believe). That would be preferable to avoid magic numbers in the inline asm :)